assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -114,6 +117,19 @@ const OverflowButtons = styled.div`
   }
 `;
 
+const buttonDimensions = { width: '24', height: '24' };
+
+const StyledGridViewButton = styled(GridViewButton).attrs(buttonDimensions)``;
+
+const SafeZoneButton = styled(SafeZone).attrs(buttonDimensions)`
+  ${({ active }) =>
+    active &&
+    css`
+      background: rgba(255, 255, 255, 0.1);
```

Comment:
I've noticed that when opacity is necessary here doing something like this is pretty common
````
import { rgba } from 'polished';
background-color: ${({ theme }) => rgba(theme.colors.bg.v13, 0.1)};
```